_includes/nav.html

```diff
@@ -20,12 +20,13 @@
       {% endfor %}
     </ul>
     <ul class="nav navbar-nav navbar-right">
-      <li><a href="https://voxpupuli.org/privacy-policy"> Privacy Policy</a></li>
+      <li><a href="/privacy-policy"> Privacy Policy</a></li>
```

Comment:
Should this also use the site.url or site.baseurl?

Reply:
We can, but this trend to be cumbersome.  I wondered if trimming `{{ site.url }}{{ site.baseurl }}` everywhere (and also the `{{ site.url }}` where `{{ site.baseurl }}` was forgotten) would make more sense, and then saw that Jekyll 4 has a `link` function that raise an error when we link to something that does not exist: https://jekyllrb.com/docs/liquid/tags/#link

Unfortunately we are running an older version of Jekyll ATM (github-pages require jekyll 3.9.3), but it would be a nice move in the future, so I will put '{{ site.url }}{{ site.baseurl }}' everywhere so that we can change it when we can.

Reply:
Done